feature/src/main/java/com/matzip/feature/base/BaseViewModel.kt

```
@@ -0,0 +1,62 @@
+            is ApiState.Error -> {
+                //TODO 에러 코드를 다룰 예정입니다.
+//                if(response.errorCode == StatusCode.ERROR_404 ||
+//                    response.errorCode == StatusCode.ERROR ||
+//                    response.errorCode == StatusCode.NETWORK_ERROR) {
+//                    showCommonError(response.errorCode)
+//                }
+//                else errorCallback?.invoke(response.errorCode)
+//                endLoading()
+            }
+            // TODO Loading 상태일때 화면을 처리할 예정입니다.
+            ApiState.Loading -> if(needLoading) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyIfBlock)